pyanaconda/modules/common/versions.py

```diff
@@ -0,0 +1,31 @@
+from functools import total_ordering
+import rpm
+
+
+@total_ordering
+class RPMVersion:
+    def __init__(self, version_string):
+        # TODO do we need to parse epoch? nothing in the tests indicates that
+        self._version, _, self._release = version_string.partition('-')
```

Comment:
Why use `.partition` and then throw away the separator? Why not just use `.split` instead?

Reply:
I always forget split takes a maxsplit argument.

Reply:
About multiple occurrences: this now works with both simpe versions without release and also with dashes is the release, so it requires no input validation.

Reply:
Using split would force me to specialcase release-less versions, such as  https://github.com/rhinstaller/anaconda/blob/055d767827046504b15b02d46ba1bf0632b4d7fa/pyanaconda/modules/storage/checker/utils.py#L240